packages/webchat/package.json

```diff
@@ -17,14 +17,14 @@
     "@botpress/messaging-components": "*",
     "@botpress/messaging-socket": "0.0.2",
     "@juggle/resize-observer": "^3.0.2",
+    "@formatjs/intl-pluralrules": "^4.1.6",
+    "@formatjs/intl-utils": "^3.8.4",
```

Comment:
Those come from the migration guide (https://formatjs.io/docs/react-intl/upgrade-guide-3x/#migrate-to-using-native-intl-apis)